### PR TITLE
fix: enable RP bindings management for workspace admins [DET-9825]

### PR DESCRIPTION
**Pull request by hkang1**
## Description

Enable RP bindings management for users with role of `WorkspaceAdmin`

## Test Plan

- [ ] set up a user with `WorkspaceAdmin` role (but not `ClusterAdmin`)
- [ ] login with that user
- [ ] go to cluster page on cluster that has non default resource pools (non default aux/gpu pools)
- [ ] verify that the non default pool can be managed (have overflow menu button on the upper right corner)
- [ ] add that RP to a workspace
- [ ] navigate to the workspace page that the PR was bound to, go to the resource pools tab
- [ ] verify that you can remove/unbind the RP that was just added

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket
DET-9825

### CODE REVIEW

File: webui/react/src/hooks/usePermissions.ts [hunk 1]
```diff
@@ -623,7 +623,8 @@ const canManageResourcePoolBindings = ({
 }: RbacOptsProps): boolean => {
   const permitted = relevantPermissions(userAssignments, userRoles);
   return rbacEnabled
-    ? permitted.has(V1PermissionType.UPDATEMASTERCONFIG)
+    ? permitted.has(V1PermissionType.UPDATEMASTERCONFIG) ||
+        permitted.has(V1PermissionType.UPDATEWORKSPACE)
```
Comment (hkang1) [inline webui/react/src/hooks/usePermissions.ts:627]: I don't see one that is specific to RP bindings for `V1PermissionType` but let me know if this is acceptable.

Comment (jgongd) [reply webui/react/src/hooks/usePermissions.ts:627]: There is `rbacv1.PermissionType_PERMISSION_TYPE_MODIFY_RP_WORKSPACE_BINDINGS` in the backend. @eecsliu Required the same in web?

Comment (hkang1) [reply webui/react/src/hooks/usePermissions.ts:627]: oh ok I see it (`MODIFYRPWORKSPACEBINDINGS`), I'll make the changes

**Review by eecsliu — APPROVED** [review]
LGTM!

File: webui/react/src/hooks/usePermissions.ts (hunk 1 above)
Comment (hkang1) [reply webui/react/src/hooks/usePermissions.ts:627]: RESOLVED: discussed async, having Workspace Update capability is the behavior we want

**Pull request merged**